playbooks/common/openshift-cluster/upgrades/pre/verify_health_checks.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Verify Host Requirements
+  hosts: oo_all_hosts
```

Comment:
@rhcarvalho sorry i missed this. no, it looks like `oo_all_hosts` comes from `g_all_hosts` which is masters/nodes/etcd only so this should be fine. all our playbooks basically need to work the same as `byo/config.yml`